components/admin/spec/rails_helper.rb

```diff
@@ -1,7 +1,7 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
-require File.expand_path('../../config/environment', __dir__)
+require File.expand_path('../../config/environment', __FILE__)
```

Comment:
My linter before said to change this to `__dir__` but that caused some weird issues so I am changing it back.

Reply:
sgtm